finetuner/experiment.py

```diff
@@ -0,0 +1,141 @@
+from typing import Any, Dict, List, Optional, Union
+
+from docarray import DocumentArray
+
+from finetuner.client import FinetunerV1Client
+from finetuner.constants import (
+    CONFIG,
+    CREATED_AT,
+    DATA,
+    DESCRIPTION,
+    EVAL_DATA,
+    MODEL,
+    NAME,
+    TRAIN_DATA,
+)
+from finetuner.hubble import push_data_to_hubble
+from finetuner.names import get_random_name
+from finetuner.run import Run
+
+
+class Experiment:
+    """Class for an experiment.
+
+    :param client: Client object for sending api requests.
+    :param name: Name of the experiment.
+    :param status: Status of the experiment.
+    :param created_at: Creation time of the experiment.
+    :param description: Optional description of the experiment.
+    """
+
+    def __init__(
+        self,
+        client: FinetunerV1Client,
+        name: str,
+        status: str,
+        created_at: str,
+        description: Optional[str] = '',
+    ):
+        self._client = client
+        self._name = name
+        self._status = status
+        self._created_at = created_at
+        self._description = description
+
+    def get_name(self):
+        """Get name of the experiment."""
+        return self._name
+
+    def get_run(self, name: str) -> Run:
+        """Get a run by its name.
+
+        :param name: Name of the run.
+        :return: A `Run` object.
+        """
+        run_info = self._client.get_run(experiment_name=self._name, run_name=name)
+        run = Run(name=run_info[NAME], experiment_name=self._name, client=self._client)
```

Comment:
this will fail here, you are omitting required arguments

Reply:
yes, I found these when I started testing. I'll change it in the next PR.